tests/unit/test_exceptions.py

```diff
@@ -38,86 +38,55 @@ def run(self, *args, **kwargs):
         with mock.patch('flask.jsonify', dummyFlask):
             super(TestExceptionHandler, self).run(*args, **kwargs)
 
-    def testMappedException(self):
```

Comment:
The essence of this test would be good to keep around -- that every exception that we define can be successfully constructed.